app/views/layouts/internal.html.erb

```diff
@@ -11,6 +11,11 @@
   <%= favicon_link_tag SiteConfig.favicon_url %>
   <title><%= controller_name.titleize %></title>
 
+  <!-- StimulusJS -->
+  <%= javascript_pack_tag "manifest", defer: true %>
+  <%= javascript_pack_tag "vendor", defer: true %>
```

Comment:
how big is vendor? I wonder if we could have a `internal-vendor` separated from the general `vendor`, not for this PR, but as a general idea

Reply:
vender seems to be 3.8MB

Reply:
It looks like it's 147kb gzipped in production...............

Which is way way way more than it should be. I had no idea.

Chart.js is most of the weight at 118kb.

We can deal with this by using dynamic import for chart.js eh?

https://dev.to/ben/we-reduced-our-vendorjs-from-210kb-to-16kb-in-about-five-minutes-of-work-and-ten-lines-of-code-1ole

Compared to chart.js, stimulus is pretty minimal, so I'd focus there first for this issue.

Reply:
Can you open a issue on that? It would be great if we incorporated one of those tools that tell us how big the packs are. 

@benhalpern I think you have the best overall picture of what our performance budget should look like - see Addy's post here https://addyosmani.com/blog/performance-budgets/ - which also suggests https://github.com/siddharthkp/bundlesize as a tool to help with this issue

Reply:
So anyway, I agree that it makes sense to have an internal vendor if, in fact, we're only using Stimulus on internal.

I made the issue about perf budget check...

https://github.com/thepracticaldev/dev.to/issues/5577

And PR for Chart.js issue...

https://github.com/thepracticaldev/dev.to/pull/5576